app/lib/pdf_filler/id39r_additional_dependents_pdf.rb

```diff
@@ -0,0 +1,32 @@
+module PdfFiller
+  class Id39rAdditionalDependentsPdf
```

Comment:
this pdf filler was added in additional to the first ID39R pdf filler because additional ID39R are only used to handle additional dependents while the first ID39R filled out will have other info